src/clarity-angular/forms/_forms.clarity.scss

```diff
@@ -8,7 +8,7 @@
 @import "../utils/layers.clarity";
 @import "../images/icons.clarity";
 
-$styledInputs: inputs("text", "password", "number", "email", "url");
+$styledInputs: inputs("text", "password", "number", "email", "url", "tel", "date");
```

Comment:
Only tested in Chrome. I'll grab screen captures of the rest and post them here.